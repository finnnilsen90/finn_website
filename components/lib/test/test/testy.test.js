const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Test = require('../test.js');

let newTest = new Test();

describe('component connection', () => {

    it('should be defined', () => {
        expect(Test).toBeDefined();
    })

})  
    
describe('<Test />', () => {

    const wrapper = render(<Test />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})

// const clickFn = jest.fn(() => 1);
// console.log(clickFn);
describe('Test', () => {
  it('button click should return 1', () => {
    const component = shallow(<Test value='1' />);
    
    expect(component).toMatchSnapshot();
    component.find('div.test').simulate('click');
    expect(component).toMatchSnapshot();
  });
});