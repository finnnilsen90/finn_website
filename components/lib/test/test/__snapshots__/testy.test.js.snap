// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> renders components 1`] = `
<div
  class="test_Container"
>
  <div
    class="test"
  >
    0
  </div>
</div>
`;

exports[`Test button click should return 1 1`] = `
<div
  className="test_Container"
>
  <div
    className="test"
  >
    0
  </div>
</div>
`;

exports[`Test button click should return 1 2`] = `
<div
  className="test_Container"
>
  <div
    className="test"
  >
    0
  </div>
</div>
`;
